bundles/CoreBundle/Migrations/Version20220119082511.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20220119082511 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Use foreign key for delete cascade on gridconfig_favourites & gridconfig_shares';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE `gridconfig_favourites`
+        CHANGE `gridConfigId` `gridConfigId` int(11) NOT NULL,
```

Comment:
I'm sorry, I used too few spaces :/ 
This is right:
```suggestion
            CHANGE `gridConfigId` `gridConfigId` int(11) NOT NULL,
```